src/applications/mhv/medical-records/actions/pagination.js

```diff
@@ -0,0 +1,39 @@
+import { Actions } from '../util/actionTypes';
+
+/**
+ * Sets Pagination in reducer where page needs to be modified by it's domain.
+ */
+export const setPagination = (domain, value) => async dispatch => {
+  dispatch({
+    type: Actions.Pagination.SET_PAGINATION,
+    payload: { domain, value },
+  });
+};
+
+/**
+ * Resets Pagination in reducer where page needs to be modified by it's domain.
+ */
+export const resetPagination = domain => async dispatch => {
```

Comment:
It looks like you are actually passing in the pathname here and then figuring out the domain. From your variable names, someone might think that you are passing in the domain and getting the url.